packages/fluent-theme/src/components/sendBox/SendBox.tsx

```diff
@@ -50,7 +50,7 @@ function SendBox(props: Props) {
   const sendMessage = useSendMessage();
   const makeThumbnail = useMakeThumbnail();
   const errorMessageId = useUniqueId('sendbox__error-message-id');
-  const [errorRef, errorMessage] = useSubmitError({ message, attachments });
+  const [errorRef, errorMessage, checkError] = useSubmitError({ message, attachments });
```

Comment:
Adding `checkError` here seems like a weird mix... I think the original idea of `useSubmitError` is to able to synchronously know what errors we have.

Then, the `checkError` makes the check asynchronous (outside of render loop). I think the `useSubmitError` would need some redesigns.

Reply:
Idea a bit changed. Maybe the name for the `checkError` is not correct... We'd like to postpone setting error on submit for most cases. It leads to better UX.

The idea was to provide both outputs to the component:
- the actual error, so `onSubmit` could rely on it
- the error we'd like to show

The `checkError` synchronizes these two states.